lib/classes/checkingaccount_class.rb

```diff
@@ -0,0 +1,47 @@
+module BankAccount
+
+  class CheckingAccount < Account
+    attr_reader :checks
+
+    #inherits passed in parameters from Account class
+    def initialize(id, initial_balance)
+    super(id, initial_balance)
+    @current_balance = initial_balance
+    @checks = 0
+    end
+
+    # used super to inherit functionality for overdraft
+    # may not be the best use of super since still needed the if/else block for the transaction fee
+    # could potentially refactor Account #withdraw to use a variable that accounts for fees/minimum balances
+    # but seems like it may be a little more complicated than needed
```

Comment:
Thanks for this thought process write out! I wasn't sure if I was being completely redundant so I'm glad I wasn't way too off base. I appreciate your review!